docs/designers-developers/glossary.md

```diff
@@ -50,7 +50,7 @@
 <dd> A type of block where the content of which is known at the time of saving a post. A static block will be saved with HTML markup directly in post content. </dd>
 
 <dt> TinyMCE </dt>
-<dd> [TinyMCE](https://www.tinymce.com/) is a web-based JavaScript WYSIWYG (What You See Is What You Get) editor. </dd>
+<dd> <a href="https://www.tinymce.com/">TinyMCE</a> is a web-based JavaScript WYSIWYG (What You See Is What You Get) editor. </dd>
```

Comment:
It's small but I'd prefer to stick to markdown where we can, unless there's a specific reason we need to use the HTML. I know dl/dt/dd aren't supported natively in a markdown format so that's a decent exception, but consistently using Markdown where it's supported would be good.

Reply:
(Does the parser get confused with HTML/markdown mixed in the same line? I guess that's a case where we would need the HTML here)

Reply:
Agreed, I'd prefer markdown but unfortunately the markdown inside the <dd> tag doesn't work. I had to push it up to Github to see, viewing that branch to see how it's rendered.

Reply:
Noted, thanks @mkaz!